diff --git a/common/py_utils/py_utils/cloud_storage.py b/common/py_utils/py_utils/cloud_storage.py
index 0dcffdcdea38025b76acbe31f33e4a5151d2e134..3bf0edce172b15fbb39232fccb089f7909906abd 100644
--- a/common/py_utils/py_utils/cloud_storage.py
+++ b/common/py_utils/py_utils/cloud_storage.py
@@ -31,7 +31,7 @@ logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 PUBLIC_BUCKET = 'chromium-telemetry'
 PARTNER_BUCKET = 'chrome-partner-telemetry'
-INTERNAL_BUCKET = 'chrome-telemetry'
+INTERNAL_BUCKET = 'brave-telemetry'
 TELEMETRY_OUTPUT = 'chrome-telemetry-output'
 
 # Uses ordered dict to make sure that bucket's key-value items are ordered from
